entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+TRACEE_EBPF_EXE=${TRACEE_EBPF_EXE:="/tracee/tracee-ebpf"}
+TRACEE_MOUNT_EXE=${TRACEE_MOUNT_EXE:="mount"}
+TRACEE_WEBHOOK_CONFIG=${TRACEE_WEBHOOK_CONFIG:="/tmp/tracee/integrations-config.yaml"}
+TRACEE_WEBHOOK_EXE=${TRACEE_WEBHOOK_EXE:="/tracee/falcosidekick"}
+TRACEE_RULES_EXE=${TRACEE_RULES_EXE:="/tracee/tracee-rules"}
+
+$TRACEE_MOUNT_EXE | grep -q 'debugfs on /sys/kernel/debug'
+if ! $TRACEE_MOUNT_EXE | grep -q 'debugfs on /sys/kernel/debug'; then
+  $TRACEE_MOUNT_EXE -t debugfs debugfs /sys/kernel/debug/
+fi
+
+if [ "$1" = "trace" ]; then
+	shift
+	$TRACEE_EBPF_EXE $@
+	exit
+fi
+
+if [ "$1" = "--rules" ]; then
```

Comment:
Why we use "--" here, unlike "trace"?
Also, shouldn't we accept any other flags than "--rules"?

Reply:
> Why we use "--" here, unlike "trace"?

because trace is a subcommand, and this is a flag to the root command

> shouldn't we accept any other flags than "--rules"

yes eventually, this PR just satisfies the MVP scope so we can make a first appearance

Reply:
`because trace is a subcommand, and this is a flag to the root command`
Didn't we say that `rules` will be a subcommand just like `trace` is?
Also, if we choose to have a tracee-ebpf image like I suggested in #463 maybe we can just remove the `trace` subcommand.

Reply:
> Didn't we say that rules will be a subcommand just like trace is?

I don't think so, maybe I missed something. but I don't see the point in adding a `rules` subcommand, and even if we had it, we would probably have made it the default

> Also, if we choose to have a tracee-ebpf

seems like we have swapped opinions :) initially I thought like you're suggesting (separate images), and then you suggested to use the same image with subcommand, and I think it a great idea, because we already have an image with all the tools inside, why not allow people to use those tools directly. 
We can still add another container dedicated for tracee-ebpf if you want to (I don't see why, it's just a few MB difference), but that doesn't have to mean we can't expose tracee-ebpf also from this one

Reply:
Yes, we can keep the `trace` subcommand.
I still think we need to have another image for tracee-ebpf as it is self-contained, and users who only need tracing (and forensics) capabilities, can just use it instead of the "tracee" image and have a simpler UX for their needs.
Moreover, the fact that today the difference is only just a few MB doesn't say that this will be the situation in the future - it will probably keep growing as we will add signatures and other functionality

Reply:
ok.. I didn't say we can't have another image. this is a discussion on the entrypoint of `tracee`